src/index.ts

```diff
@@ -530,10 +553,12 @@ function mapAvailability(data: {[key: string]: string}): number[] {
           activeHours.push([startTime, endTime]);
         }
       }
-      availableMonths[i] = {
+      availableMonths.push({
+        month: i,
         isAllDay,
         activeHours,
-      };
+        season: isNH ? NH_SEASON[i - 1] : SH_SEASON[i - 1],
```

Comment:
I feel like at this point `i` should be `0` and `month` should be the one to `+1`

Reply:
Agree with you. Better to +1 an int than +1 an index in an array.